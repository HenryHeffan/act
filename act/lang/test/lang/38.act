defproc testme (chan?(int<8>) A, B; chan!(bool) OUT)
{                                                      
  chan(bool) OUT1;
  chan(bool) tmp;
  int x;
  bool y;
  chp {                                          
    *[ [ #tmp -> OUT!false [] else -> skip ] ]
  }                                                    
}                                                      

testme t;
