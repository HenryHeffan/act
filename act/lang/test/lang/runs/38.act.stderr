In expanding testme (38.act:1)
In expanding ::<Global>
FATAL: Cannot have an else clause when guards contain channels
